xclim/downscaling/utils.py

```diff
@@ -118,7 +118,7 @@ def adapt_freq(
     group: str = "time",
     window: int = 1,
 ):
-    """
+    r"""
```

Comment:
If you're using raw strings, doesn't that affect the double backslashes in your latex formulas?

Reply:
sphinx chokes on formulas otherwise. Suggestion ?

Reply:
Nothing. I just wasn't sure if that was meant to be there. Good to know!